lib/manager/npm/extract/index.ts

```diff
@@ -71,23 +80,23 @@ export async function extractPackageFile(
   } else {
     yarnWorkspacesPackages = packageJson.workspaces?.packages;
   }
-  const packageJsonType = mightBeABrowserLibrary(packageJson)
+  const packageJsonType: PackageJsonType = mightBeABrowserLibrary(packageJson)
```

Comment:
```suggestion
  const packageJsonType = mightBeABrowserLibrary(packageJson)
```

Shouldn't the return type of `mightBeABrowserLibrary` aready `PackageJsonType` ?